tools/dds/dds-server/rs-dds-server.cpp

```diff
@@ -138,6 +176,18 @@ std::vector< std::shared_ptr< realdds::dds_stream_server > > get_supported_strea
             continue;
         }
         server->init_profiles( profiles, default_profile_index );
+        
+        //Set stream intrinsics
+        auto video_server = std::dynamic_pointer_cast< dds_video_stream_server >( server );
+        auto motion_server = std::dynamic_pointer_cast< dds_motion_stream_server >( server );
```

Comment:
here - both are servers, because it is casted from servers - seems the others (other comments) should be video_stream and motion_stream

Reply:
Thanks, as mentioned in other comment, copy pasted from here and corrected there